src/Features/Core/Portable/Completion/Providers/AbstractPartialMethodCompletionProvider.cs

```diff
@@ -27,11 +27,11 @@ internal abstract partial class AbstractPartialCompletionProvider : AbstractMemb
                         SymbolDisplayMiscellaneousOptions.EscapeKeywordIdentifiers |
                         SymbolDisplayMiscellaneousOptions.UseSpecialTypes);
 
-        public AbstractPartialCompletionProvider()
+        public AbstractPartialMethodCompletionProvider()
         {
         }
 
-        protected abstract bool IsPartialCompletionContext(SyntaxTree tree, int position, CancellationToken cancellationToken, out DeclarationModifiers modifiers, out SyntaxToken token);
+        protected abstract bool IsPartialMethodCompletionContext(SyntaxTree tree, int position, CancellationToken cancellationToken, out DeclarationModifiers modifiers, out SyntaxToken token);
         protected abstract string GetDisplayText(IMethodSymbol method, SemanticModel semanticModel, int position);
         protected abstract bool IsPartial(IMethodSymbol m);
```

Comment:
not part of your change.  but if you could rename this from 'm' to 'method' that would be appreciated.